validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorApiChannel.java

```diff
@@ -42,6 +43,9 @@
 
   SafeFuture<Map<BLSPublicKey, Integer>> getValidatorIndices(final List<BLSPublicKey> publicKeys);
 
+  SafeFuture<Optional<Map<BLSPublicKey, ValidatorStatus>>> getValidatorStatuses(
+      final List<String> validatorIdentifiers);
```

Comment:
Is it possible to use a more specific type for `validatorIdentifiers `?  `String` params are a bit confusing - requires you to dig down to figure out what they're supposed to hold / how they should be formatted.
```suggestion
      final List<BLSPublicKey> validatorIdentifiers);
```

Reply:
Very valid point. Done.